Linkta-org/core
# implemented styling on nodes

## Description



Implements Node and Edge Type to add appropriate styling to LinktaFlow components

Fixes # (issue)
#352  

## Type of change




## How Can this be tested? Testing Plan to review this PR




Change is in adding type to each flow component (node and edge). In order to test this change, create a new flow and ensure styling is visible. Make changes within LinktaNode.tsx in order to ensure styling is updated within the displayed LinktaFlow. Styling should match changes made

## Checklist:




## Additional Information:
